### PR TITLE
08 objects tasks

### PR DESCRIPTION
**Pull request by Saik1s**

### CODE REVIEW

File: task/08-objects-tasks.js [hunk 1]
```diff
@@ -105,36 +111,75 @@ function fromJSON(proto, json) {
...
+        this.selector = `${selector1.stringify()} ${combinator} ${selector2.stringify()}`;
+        return this;
+    }
+
+    stringify() {
+        return this.selector;
+    }
+
+    checkType(type) {
+        if (type < this.type) throw this.orderError;
```
Comment (Geass1000) [inline task/08-objects-tasks.js]: Do test allow to use `new Error` class? If answer is "yes", you have to use `new Error` class instead of `string`.

Comment (Geass1000) [inline task/08-objects-tasks.js]: JFYI: use braces in each `for`/`while`/`do ... while`/`if` operator. You will make less mistakes.

**Comment by Geass1000** [comment]
Also you must fix CI errors. Click `Details` to see logs.

**Comment by Saik1s** [comment]
CI does not support .includes, in IDE all tests passed

**Comment by Geass1000** [comment]
Well, what can I say? It is life)))

**Pull request merged**